modules/ui/core/preset.js

```diff
@@ -1,3 +1,9 @@
+import { Icon } from '../../svg/index';
+import { TagReference } from './tag_reference';
+import { Browse } from '../../modes/index';
+import { presetObj } from '../preset/index';
```

Comment:
Again, I have this naming problem. So this time I named it `presetObj`. Let me know if something better fits.